src/Sql/Sql/help/Get-AzSqlInstance.md

```diff
@@ -137,34 +311,106 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -InstancePool
+The instance pool parent object.
+
+```yaml
+Type: Microsoft.Azure.Commands.Sql.Instance_Pools.Model.AzureSqlInstancePoolModel
+Parameter Sets: ListByInstancePoolObjectParameterSet
+Aliases: ParentObject
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -InstancePoolName
+The name of the instance pool.
+
+```yaml
+Type: System.String
+Parameter Sets: ListByInstancePoolParameterSet
+Aliases:
+
+Required: True
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -InstancePoolResourceId
+The instance pool resource identifier.
+
+```yaml
+Type: System.String
+Parameter Sets: ListByInstancePoolResourceIdentiferParameterSet
+Aliases:
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
 ### -Name
 SQL instance name.
 
 ```yaml
 Type: System.String
-Parameter Sets: (All)
+Parameter Sets: GetInstanceByNameAndResourceGroup
 Aliases: InstanceName
 
-Required: False
-Position: 0
+Required: True
```

Comment:
@johnpaulkee it looks like the changes to this command have reversed the positional order of `-Name` and `-ResourceGroupName`, which is a breaking change. Would you mind changing this back?

Also, it looks like `-Name` is now mandatory in its parameter set when it was optional before -- this would be a breaking change if the user was able to previously run the command with just `-Name` and no `-ResourceGroupName`. If the command previously failed in this case, then this isn't a breaking change.

Reply:
Sure I'll revert this :)

Reply:
>  If the command previously failed in this case, then this isn't a breaking change.
Will get back to you on this.

Reply:
I can't seem to run:
![image](https://user-images.githubusercontent.com/8611517/62171396-ad84e980-b2e3-11e9-9b31-c49818bcf7a0.png)
Given this, I think it isn't a breaking change.
@cormacpayne let me know what you think.

Reply:
Weird, I just ran the same thing on Cloud Shell and it didn't prompt me for `-ResourceGroupName`. I'm not sure what would be causing it to prompt you 😕 Would you mind installing the latest `Az.Sql` module on your machine and testing it locally?

Regardless, I think the case we want to be cautious of is when the user only provided `-ResourceGroupName`, because with the new parameter set, it will prompt them to provide a value for `-Name` where it didn't previously. That would be the breaking change in this case.

Reply:
Just kidding, locally in master we can do Name by itself.
![image](https://user-images.githubusercontent.com/8611517/62174067-5edc4d00-b2ed-11e9-9f64-cf62240fb780.png)

Reply:
Updated so that Name and ResourceGroup are non mandatory again. Also removed positional values since they were both non mandatory.